chamber/models.py

```diff
@@ -22,13 +21,20 @@ def __init__(self, settings, ref='Mills', rule='mean'):
         self.temp_e = settings['temp_e']
         self.ref = ref
         self.rule = rule
-        self.temp_s = (self.temp_e + self.temp_dp) / 2
+        self.temp_s = (self.temp_e + self.temp_dp)/2
+        self.mu_m = None
+        self.cp_m = None
         self.d_12 = None
```

Comment:
I try to keep all of these in alphabetical order. When a class has many attributes (more than 8), it becomes difficult to keep track of them. If you could alphabetize all the attributes that are set to None, that would be helpful.